config_example.yaml

```
@@ -11,6 +11,7 @@ slack:
   #icon: "" # Slack icon (avatar)
   outputformat: "text" # all (default), text, fields
   minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informationnal|debug or "" (default)
+  messageformat: "Alert for rule '{{ .Rule }}' from process {{ index .OutputFields \"proc.name\" }} " # a Go template to format Slack messages, see Slack Message Formatting in the README for details
```

I propose this to have a format which works with example I give [here](https://github.com/falcosecurity/falcosidekick#examples)

```suggestion
  messageformat: "Alert : rule *{{ .Rule }}* triggered by user *{{ index .OutputFields \"user.name\" }}*" # a Go template to format Slack Text above Attachment, displayed in addition to the output from `SLACK_OUTPUTFORMAT`, see [Slack Message Formatting](#slack-message-formatting) in the README for details. If empty, no Text is displayed before Attachment.
```

I've also added a small detail about what part of Slack message is modified with using Slack's terminology.